src/settings.ts

```diff
@@ -16,25 +17,36 @@ function setFieldFromPrivateSettings(fieldIdAndSettingName: LocalSetting) {
 function setFieldFromBoardSettings(fieldIdAndSettingName: BoardSetting) {
   let value = store.getBoardSetting(fieldIdAndSettingName)
   if (value) {
-    let field = document.getElementById(fieldIdAndSettingName)
-    if (field) field["value"] = value
+    let field = document.getElementById(fieldIdAndSettingName);
+    if (field) field["value"] = value;
   }
 }
 
 function addValueOfFieldToObject(object: any, fieldId: BoardSetting | LocalSetting) {
-  let field = document.getElementById(fieldId)
-  object[fieldId] = field ? field["value"] : null
-  return object
+  let field = document.getElementById(fieldId);
+  let value = field ? field["value"] : null;
+  object[fieldId] = value;
+  return object;
+}
+
+function hashAndSaveCredentials() {
+  let username = (document.getElementById(LocalSetting.CB_USERNAME) as HTMLInputElement).value;
+  let password = (document.getElementById(LocalSetting.CB_PASSWORD) as HTMLInputElement).value;
+
+  const ha1 = md5(`${username}:CodeBeamer:${password}`);
+
+  localStorage.setItem('cb_auth_hash', ha1);
```

Comment:
Why exactly is this stored in the localStorage while the other credentials are stored in the sessionStorage?

Reply:
It's not done so delibaretly, but is merely a remnant of an abandoned feature (digest auth). The function is not used anywhere.
That's also why I'm just using some string instead of another value in the `LocalSetting` enum.